src/widgets/_index.scss

```diff
@@ -25,6 +25,23 @@ messagedialog,
 @define-color theme_selected_fg_color #{"" + $fg-color};
 @define-color wm_shadow #{rgba(black, 0.2)};
 
+// Things other apps expect
+@define-color theme_text_color #{'@text_color'};
+@define-color theme_bg_color #{'@bg_color'};
+@define-color theme_base_color #{'@base_color'};
```

Comment:
I don't think this is defined yet

Reply:
I'll define it as `bg-color(1)` to match previous styling on old stylesheet.